File: phongs_work.js
// fibonacci function

let fibonacci = function(n) {
    if (n <= 1) return 1;
    return fibonacci(n - 1) + fibonacci(n - 2);
}

console.log("Results:");
for (let i = 0; i < 15; i++) {
    console.log(fibonacci(i));
}

File: phongs_work.py
def fibonacci(n):
    if n in [0, 1]:
        return n
    return fibonacci(n - 1) + fibonacci(n - 2)


def main():
    results = [fibonacci(n) for n in range(15)]
    print(f"Results: {results}")


if __name__ == "__main__":
    main()

